clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -98,13 +98,12 @@
     private int sessionTimeoutMs = 10000;
     private int heartbeatIntervalMs = 5000;
     private long retryBackoffMs = 100;
-    private boolean autoCommitEnabled = false;
+    private boolean autoCommitEnabled = true;
```

Comment:
We intentionally disable auto-commit so that it does not interfere with other test cases. If we are testing auto-commit behavior in a test, we just enable it locally for that test case (see `testAutoCommitDynamicAssignment` for example). Let's continue to follow this pattern.